armi/physics/fuelCycle/translationFunctions.py

```diff
@@ -0,0 +1,676 @@
+import json
```

Comment:
This file is only 65% covered by unit tests.

https://coveralls.io/builds/51800037/source?filename=armi%2Fphysics%2FfuelCycle%2FtranslationFunctions.py

Can we add more tests to cover this?

Reply:
Added alot more testing. Let me know!